server/server.go

```
@@ -33,25 +40,31 @@ func NewServer(cfg Config, queryable querier.Querier, logger *slog.Logger) (*ser
-	// Fetching logs and sources
+	// API
+	mux.HandleFunc("GET /api/healthcheck", s.healthCheckHandler)
 	mux.HandleFunc("POST /api/logs/search", s.searchLogsHandler)
 
 	return s.recoverPanicMiddleware(s.requestLoggerMiddleware(s.corsMiddleware(mux)))
 }
 
 func (s *server) Serve(ctx context.Context) error {
+	s.startTime = time.Now()
+
 	srv := &http.Server{
```

golangci-lint (2.11.4)

[medium] G112: Potential Slowloris Attack because ReadHeaderTimeout is not configured in the http.Server

(gosec)